backend/src/components/organizations.js

```diff
@@ -26,7 +26,35 @@ async function getOrganizationFacilities(req, res) {
   }
 }
 
+async function getOrganizationUsers(req, res) {
+  try {
+    const { firstName, lastName, email } = req.query
+    let filterCriteria = [`parentcustomerid_account/accountid eq '${req.params.organizationId}'`]
+    if (firstName) {
+      filterCriteria.push(`ofm_first_name eq '${firstName}'`)
+    }
+    if (lastName) {
+      filterCriteria.push(`ofm_last_name eq '${lastName}'`)
+    }
+    if (email) {
+      filterCriteria.push(`emailaddress1 eq '${email}'`)
+    }
+    const operation = `contacts?$select=contactid,ccof_username,ofm_first_name,ofm_last_name,emailaddress1&$filter=${filterCriteria.join(' and ')}`
+    const response = await getOperation(operation)
+    if (!response?.value) {
+      return res.status(HttpStatus.NOT_FOUND).json({ error: 'No users found' })
+    }
```

Comment:
I don't know that a 404 is the proper response for an empty query, as opposed to say /users/someuserthatisntreal. I think a 200 and empty array is sufficient. It will also simplify your code.

Reply:
Removed.